README.rst

```diff
@@ -5,17 +5,21 @@ ContinuousFlex plugin
 This plugin provides the latest Scipion protocols for cryo-EM continuous conformational flexibility/heterogeneity analysis of biomolecular complexes.
 
 
-Installation
+Requirements
 ------------
 
-You will need to use `3.0 <https://github.com/I2PC/scipion/releases>`_ version of Scipion to be able to run these protocols. To install the plugin, you have two options:
-We you need help installing Scipion3, please refer to the Scipion Documentation `here <https://scipion-em.github.io/docs/docs/scipion-modes/how-to-install.html>`__
+You will need to use `3.0 <https://github.com/I2PC/scipion/releases>`_ version of Scipion to be able to run these protocols.
+If you need help installing Scipion3, please refer to the Scipion Documentation `here <https://scipion-em.github.io/docs/docs/scipion-modes/how-to-install.html>`__
 
 Make sure that you have cmake installed on your Linux system. For example, if you are using Ubuntu
  .. code-block::
 
     sudo apt install cmake
 
+Installation
+------------
+
+To install the plugin, you have two options:
 
 a) Stable version
 
```

Comment:
Since genesis is a new thing, when someone installs the plugin in devel mode we should give them the instructions to install it in devel as well.
I think this command will do the trick:
`scipion3 installb genesis`